Mathlib/FieldTheory/IsAlgClosed/AlgebraicClosure.lean

```diff
@@ -201,3 +201,28 @@ instance {L : Type*} [Field k] [Field L] [Algebra k L] [Algebra.IsAlgebraic k L]
   isAlgClosed := inferInstance
 
 end AlgebraicClosure
+
+namespace IntermediateField
+
+variable {K L : Type*} [Field K] [Field L] [Algebra K L] (E : IntermediateField K L)
+
+theorem AdjoinSimple.is_algebraicClosure {x : L} (hx : IsIntegral K x) :
+    IsAlgClosure K (AlgebraicClosure ↥K⟮x⟯) :=
+  have : Algebra.IsAlgebraic K K⟮x⟯ := (isAlgebraic_adjoin_simple hx)
+  ⟨AlgebraicClosure.isAlgClosed ↥K⟮x⟯,
+    Algebra.IsAlgebraic.trans K K⟮x⟯ (AlgebraicClosure ↥K⟮x⟯)⟩
```

Comment:
I think this should work for every algebraic extension of `K`, and could be an instance.

Reply:
Good point, thanks!